<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Point_Widget</class>
 <widget class="QWidget" name="Point_Widget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>468</width>
    <height>550</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>468</width>
    <height>0</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="mainVerticalLayout">
   <item>
    <widget class="QTableView" name="tableView"/>
   </item>
   <item>
    <widget class="Section" name="export_section" native="true">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <layout class="QVBoxLayout" name="exportSectionLayout">
      <property name="leftMargin">
       <number>0</number>
      </property>
      <property name="topMargin">
       <number>0</number>
      </property>
      <property name="rightMargin">
       <number>0</number>
      </property>
      <property name="bottomMargin">
       <number>0</number>
      </property>
      <item>
       <layout class="QHBoxLayout" name="exportTypeLayout">
        <item>
         <widget class="QLabel" name="label_export_type">
          <property name="text">
           <string>Export Type:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QComboBox" name="export_type_combo">
          <item>
           <property name="text">
            <string>CSV</string>
           </property>
          </item>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QStackedWidget" name="stacked_saver_options">
        <property name="currentIndex">
         <number>0</number>
        </property>
        <widget class="CSVPointSaver_Widget" name="csv_point_saver_widget"/>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="export_media_frames_checkbox">
        <property name="text">
         <string>Export matching media frames</string>
        </property>
        <property name="checked">
         <bool>false</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="MediaExport_Widget" name="media_export_options_widget" native="true"/>
      </item>
      <item>
       <widget class="QCheckBox" name="propagate_checkbox">
        <property name="text">
         <string>Propagate</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>MediaExport_Widget</class>
   <extends>QWidget</extends>
           <header>MediaExport/MediaExport_Widget.hpp</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>CSVPointSaver_Widget</class>
   <extends>QWidget</extends>
   <header>IO_Widgets/Points/CSV/CSVPointSaver_Widget.hpp</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>Section</class>
   <extends>QWidget</extends>
   <header>Collapsible_Widget/Section.hpp</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
